scout/constants/acmg.py

```diff
@@ -60,22 +60,24 @@
                     (
                         "PS1",
                         {
-                            "short": "Known pathogenic aa (HQ)",
+                            "short": "Known pathogenic aa",
```

Comment:
The meaning of HQ is unclear to me here. Perhaps the intention was to disambiguate it from PM5 or PP5? Regardless, unless you know, let's remove it.

Reply:
I have no idea, let's remove it!